packages/core/src/api/channel.ts

```diff
@@ -92,9 +92,17 @@ export class ChannelsAPI {
 	 * @see {@link https://discord.com/developers/docs/resources/message#get-reactions}
 	 * @param channelId - The id of the channel the message is in
 	 * @param messageId - The id of the message to get the reactions for
-	 * @param emoji - The emoji to get the reactions for
+	 * @param emoji - The emoji to get the reactions for. Encoding happens internally
```

Comment:
I'd make it clearer, what kind of encoding?
```suggestion
	 * @param emoji - The emoji to get the reactions for. URL encoding happens internally
```

Reply:
Resolved with a3fc5bf40befa9a65efa3ccb7925c5b678b796a5.